bin/data-symlinks

```diff
@@ -2,14 +2,14 @@
 
 set -e -u -o pipefail
 
-data=$1
+data="${1:-data}"
```

Comment:
This program is intended to be run from `make data-update`, not by hand.
Better might be:
```
data=${1:?Don't run this by hand. Use 'make data-update'}
```

Bash doesn't need quotes for the RHS of assignments unless there spaces or syntax characters.
Even assignments like `f=$(ls -a "$dir")` doesn't need quotes because Bash parsing knows it is an expression and isn't confused by whitespace in `$(...)` (but vars in command args always need to be quoted).

Reply:
> This program is intended to be run from make data-update, not by hand.

It's possible to run it by hand, for example when developing, I don't want to run the whole data-update process over and over if I only want to recreate the symlinks.
I don't see what's bad about setting `data` as default.

Reply:
OK, but remove the double quotes here.

Reply:
done